src/calculateRentalCost.js

```diff
@@ -4,7 +4,25 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCOUNT = 50;
+
+  const SHORT_TERM = 3;
+  const SHORT_TERM_DISCOUNT = 20;
+
+  const DAY_PRICE = 40;
+
+  const PRICE = days * DAY_PRICE;
```

Comment:
```suggestion
  const currentPrice = days * DAY_PRICE;
```
Variables should be in camelCase